shared/fs/popups/row-action-popup.js

```diff
@@ -32,16 +32,32 @@ const Popup = (props: Props) => {
     view: (
       <Box style={stylesHeader}>
         <PathItemIcon spec={props.itemStyles.iconSpec} style={pathItemIconStyle} />
-        <Text type="BodySmallSemibold" style={{color: props.itemStyles.textColor}} lineClamp={1}>
-          {props.name}
-        </Text>
+        <Box style={stylesNameTextBox}>
+          <Text
+            type="BodySmallSemibold"
+            style={
+              // $FlowFixMe
+              {
+                color: props.itemStyles.textColor,
+                overflowWrap: 'break-word',
```

Comment:
This is only supported by `text.desktop.js`. Will I have to break this file into .desktop and .native just for this, or is there a better way to do it?

Reply:
You could do:
```javascript
{
  color: ...,
  textAlign: ...,
  ...(isMobile ? {} : {overflowWrap: 'break-word'}),
}
```

Reply:
I think using `platformStyles` is the preferred way to handle these

Reply:
Not sure how I didn't think of trying those. I'm kind of embarrassed for asking the question 😂